starburst/etc/catalog/mssql.properties

```diff
@@ -1,4 +1,4 @@
 connector.name=sqlserver
-connection-url=jdbc:sqlserver://mssql:1433;databasename=Test_EMIS
+connection-url=jdbc:sqlserver://mssql:1433;database=Test_EMIS
```

Comment:
I'm not sure! It may not be required. This was added in the "let's try anything" stage of getting this to work.

Reference:

https://trino.io/docs/current/connector/sqlserver.html#configuration